### PR TITLE
refactor(deepl): rewrite

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title
image
image

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Review by isabelroses — COMMENTED** [review]
This seems to have broken a few things
image
image
image

Also the glossary button but i can't seem to screenshot that, and the mic hover too

**Comment by somerand0mcat** [comment]
the mic hover works on my end
image

**Comment by somerand0mcat** [comment]
anyways i fixed the issues except that "allow mic" prompt which didn't want to theme and also i dont understand what you mean on the third screenshot

**Comment by isabelroses** [comment]
> same for that "glossary button" too

image
This button here appears distorted, and when you click it the popup ins unthemed

**Comment by isabelroses** [comment]
Looks a lot better :+1: nothing too noticeable too me other then some accents. And that weird box from the first screenshot.

**Comment by somerand0mcat** [comment]
> Looks a lot better 👍 nothing too noticeable too me other then some accents. And that weird box from the first screenshot.

i found out that i accidentally filtered the box which is why its broken
i'll fix it later because i dont have much time right now

**Review by watatomo — APPROVED** [review]

File: styles/deepl/catppuccin.user.css [hunk 1]
```diff
@@ -72,675 +70,283 @@
...
-        place-content: center center;
-        align-items: center;
-
-        > div:first-of-type {
-          width: 100vw;
-        }
-      }
+    .Logo-module--logoImg--1ee7e, .deeplLogo-module--logo--ffd7a {
+        filter: invert(61%) sepia(85%) saturate(375%) hue-rotate(189deg) brightness(98%) contrast(100%);
+     }
```
Comment (uncenter) [inline styles/deepl/catppuccin.user.css:349]: Please don't do this. Add a variable to the palette at the bottom like we do with other userstyles, and add then add a variable declaration at the top below the others. I didn't get to this PR in time so please do this in a followup PR.

Comment (uncenter) [reply styles/deepl/catppuccin.user.css:349]: E.g. https://github.com/search?q=repo:catppuccin/userstyles+%22filter:+@%22&type=code.

Comment (somerand0mcat) [reply styles/deepl/catppuccin.user.css:349]: > Please don't do this. Add a variable to the palette at the bottom like we do with other userstyles, and add then add a variable declaration at the top below the others. I didn't get to this PR in time so please do this in a followup PR.

i did,  #1109

**Pull request merged**